src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java

```diff
@@ -1,17 +1,23 @@
 package com.nttdata.nttdatanotificationservice.rocket.models;
 
 public class RocketMessage {
+
+    private static final String AVATAR = "https://user-images.githubusercontent.com/51387119/82707419-ddb1c600-9c30-11ea-8bfa-b3c624b23cdd.png";
```

Comment:
duplicated, you should move that property up, both teams and rocket message are using it

Reply:
Refactored, take a look and tell me if that is what you meant